src/services/Initialization.cpp

```diff
@@ -3,13 +3,17 @@
 #include "services\ServiceLocator.hh"
 #include "services\impl\JsonFileConfiguration.hh"
 #include "services\impl\LeaderboardManager.hh"
+#include "services\impl\WindowsFileSystem.hh"
 
 namespace ra {
 namespace services {
 
-void Initialization::RegisterServices(const std::wstring& sHomeDir, const std::string& sClientName)
+void Initialization::RegisterServices(const std::string& sClientName)
 {
-    std::wstring sFilename = sHomeDir + L"RAPrefs_" + ra::Widen(sClientName) + L".cfg";
+    auto* pFileSystem = new ra::services::impl::WindowsFileSystem();
```

Comment:
You don't have to put `* `for pointers, but if that's what you prefer go ahead. Anyway, are these pointers being managed somewhere?

Reply:
`ServiceLocator::Service` has a `unique_ptr` for managing the instance.

Reply:
the construct is typically written as:
```
ServiceLocator::Provide<Interface>(new Implementation);
```
the allocation is more obvious when on a separate line.